### PR TITLE
Bump version for release

### PR DESCRIPTION
**Pull request by jaymzh**
Signed-off-by: Phil Dibowitz <phil@ipom.com>

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,18 @@
+## 0.0.14
+* Always use the desired knife config
+* Robocop compliance
```
Comment (davide125) [inline CHANGELOG.md]: s/Robocop/Rubocop/

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,18 @@
...
+* Add "partial failure" reporting
+* Fix race conditions
+* Better detection for existing chef-zero instances
+* README corrections
+* noop transport fixes
```
Comment (davide125) [inline CHANGELOG.md]: s/noop/NoOp/

Comment (jaymzh) [reply CHANGELOG.md]: The *class* is NoOp, but the argument to `--transport` is `noop`.

**Comment by jaymzh** [comment]
I thought impact and profile were the same. Added impact.

File: CHANGELOG.md (hunk 1 above)
Comment (jaymzh) [reply CHANGELOG.md]: done

**Pull request merged**